src/histolab/filters/image_filters.py

```diff
@@ -17,16 +17,45 @@
 # ------------------------------------------------------------------------
 
 import operator
-from typing import Any, Callable, List, Union
+from abc import abstractmethod
+from typing import Callable, List, Union
 
 import numpy as np
 import PIL
 
 from .. import util as U
```

Comment:
I know that you didn't touch this line, but i wonder why we used a short name like `U` for the util module. Considering that you're touching this file in this PR i'd ask you to change `U` and import and use `utils` as is.

Reply:
done